test/db/archos/linux-x64/dbg

```diff
@@ -95,3 +95,13 @@ EXPECT_ERR=<<EOF
 child received signal 9
 EOF
 RUN
+
+NAME=rizin -d no error
+FILE=--
+CMDS=<<EOF
+envDISPLAY=:0
```

Comment:
why no space after `env`?

Reply:
as per below ... surprisingly it does work

![env-help](https://github.com/rizinorg/rizin/assets/12002672/c653336d-b094-4fce-8ed8-1436a92c0e04)

Reply:
hmm maybe i shoudn't depend too much on typos

Reply:
9a93959

Reply:
Yeah that's an unexpected behaviour due to the fact `env` was `%` before. Should be fixed, creating an issue now!